### ISSUE TITLE
Connect on Chrome OS

### ISSUE THREAD

**Issue by CMDann** [googlecreativelab/coder#31]
For some reason I am totally unable to connect to Coder on my Chromebook. I can connect totally fine on my Mac Mini and Windows desktop. I am given an error about DNS. Is this because of a way that the Chrome OS handles DNS as opposed to how it works on the other desktops or is this just a weird bug for me?

**Comment by inder-faichi** [comment]
I had same issue and I fixed it by changing my preferred DNS server to 8.8.8.8 in windows.

**Comment by zbintliff** [comment]
I know this is an older issue but I am also facing it.  Able to connect via hostname from all other devices except chromebook.  Can access via ip, but not the local hostname.

**Comment by CVBruce** [comment]
Chrome books/boxes lack the mDNS software used to discover your Pi.  On your Pi you are running Avahi which provides that function.

**Comment by zbintliff** [comment]
@CVBruce yeah i found the answer elsewhere and just ended up changing my /etc/hosts.

Thanks for the info.

**Comment by heddmorf** [comment]
There are feature requests and bugs in the Chrome OS to make it work.

If you have a google account, log in and star these: original [feature request](https://bugs.chromium.org/p/chromium/issues/detail?id=199397) and current [bug](https://bugs.chromium.org/p/chromium/issues/detail?id=626377).

Currently less than 100 people have 'starred' the issue, so the developers think we don't care too much about fixing it.
